docs/sources/installation/security.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Security"
+description = "Security Docs"
+keywords = ["grafana", "security", "documentation"]
+type = "docs"
+[menu.docs]
+name = "Security"
+identifier = "security"
+parent = "admin"
+weight = 2
++++
+
+# Security
+
+## Data source proxy and insecure internal web services
```

Comment:
```suggestion
## Data source proxy and protecting internal services
```

I think the headline poses questions about if the internal services are in Grafana or somewhere else. I'd rather put a positive spin on it and have the text explain what it's about.